### PR TITLE
cloud: categorize FAQs

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/tidb-cloud-faq.md [hunk 1]
```diff
@@ -69,42 +67,58 @@ TiKV is the transactional store used to store OLTP data. All the data in TiKV is
...
+### How does TiDB replicate data between the TiKV nodes?
 
 TiKV divides the key-value space into key ranges, and each key range is treated as a “Region”. In TiKV, data is distributed among all nodes in a cluster and uses the Region as the basic unit. PD is responsible for spreading (scheduling) Regions as evenly as possible across all nodes in a cluster.
 
 TiDB uses the Raft consensus algorithm to replicate data by Regions. Multiple replicas of a Region stored in different nodes form a Raft Group.
 
 Each data change is recorded as a Raft log. Through Raft log replication, data is safely and reliably replicated to multiple nodes of the Raft Group.
 
-## How do I make use of TiDB Cloud’s HTAP capabilities?
+## High Availability FAQs
```
Comment (TomShawn) [inline tidb-cloud/tidb-cloud-faq.md:78]:
```suggestion
## High Availability FAQ
```

File: tidb-cloud/tidb-cloud-faq.md [hunk 2]
```diff
@@ -69,42 +67,58 @@ TiKV is the transactional store used to store OLTP data. All the data in TiKV is
...
+TiDB uses the Raft consensus algorithm to ensure that data is highly available and safely replicated throughout storage in Raft Groups. Data is redundantly copied between TiKV nodes and placed in different Availability Zones to protect against machine or data center failure. With automatic failover, TiDB ensures that your service is always on.
 
-For further HTAP scenarios, refer to [How We Build an HTAP Database That Simplifies Your Data Platform](https://pingcap.com/blog/how-we-build-an-htap-database-that-simplifies-your-data-platform).
+As a Software as a Service (SaaS) provider, we take data security seriously. We have established strict information security policies and procedures required by the [Service Organization Control (SOC) 2 Type 1 compliance](https://pingcap.com/blog/pingcap-successfully-completes-soc-2-type-1-examination-for-tidb-cloud). This ensures that your data is secure, available, and confidential.
```
Comment (TomShawn) [inline tidb-cloud/tidb-cloud-faq.md:84]:
```suggestion
As a Software as a Service (SaaS) provider, we take data security seriously. We have established strict information security policies and procedures required by the [Service Organization Control (SOC) 2 Type 1 compliance](https://en.pingcap.com/press-release/pingcap-successfully-completes-soc-2-type-1-examination-for-tidb-cloud/). This ensures that your data is secure, available, and confidential.
```

File: tidb-cloud/tidb-cloud-faq.md [hunk 3]
```diff
@@ -69,42 +67,58 @@ TiKV is the transactional store used to store OLTP data. All the data in TiKV is
...
+
+### Is there an easy migration path from another RDBMS to TiDB Cloud?
 
 TiDB is highly compatible with MySQL. You can migrate data from any MySQL-compatible databases to TiDB smoothly, whether the data is from a self-hosted MySQL instance or RDS service provided by the public cloud. For more information, see [Migrate Data from MySQL-Compatible Databases](/tidb-cloud/migrate-data-into-tidb.md).
 
-## How does TiDB protect data privacy and ensure security?
+## Feature FAQs
 
-Transport Layer Security (TLS) and Transparent Data Encryption (TDE) are included for encryption at rest. There are two different network planes: the application to the TiDB server and the plane for data communication. We include extended syntax to compare Subject Alternative Name for verification of certificates and TLS context for internal communication.
+### How do I make use of TiDB Cloud’s HTAP capabilities?
```
Comment (TomShawn) [inline tidb-cloud/tidb-cloud-faq.md:94]:
```suggestion
### How do I make use of TiDB Cloud's HTAP capabilities?
```

File: tidb-cloud/tidb-cloud-faq.md [hunk 4]
```diff
@@ -69,42 +67,58 @@ TiKV is the transactional store used to store OLTP data. All the data in TiKV is
...
 
-## Is there an easy migration path from another RDBMS to TiDB Cloud?
+## Migration FAQs
+
+### Is there an easy migration path from another RDBMS to TiDB Cloud?
 
 TiDB is highly compatible with MySQL. You can migrate data from any MySQL-compatible databases to TiDB smoothly, whether the data is from a self-hosted MySQL instance or RDS service provided by the public cloud. For more information, see [Migrate Data from MySQL-Compatible Databases](/tidb-cloud/migrate-data-into-tidb.md).
 
-## How does TiDB protect data privacy and ensure security?
+## Feature FAQs
```
Comment (TomShawn) [inline tidb-cloud/tidb-cloud-faq.md:92]: This is a bit confusing to me.

File: tidb-cloud/tidb-cloud-faq.md [hunk 5]
```diff
@@ -69,42 +67,58 @@ TiKV is the transactional store used to store OLTP data. All the data in TiKV is
...
+### How does TiDB protect data privacy and ensure security?
+
+Transport Layer Security (TLS) and Transparent Data Encryption (TDE) are included for encryption at rest. There are two different network planes: the application to the TiDB server and the plane for data communication. We include extended syntax to compare Subject Alternative Name for verification of certificates and TLS context for internal communication.
+
+### Can TiDB Cloud run in our VPC?
 
 No. TiDB Cloud runs on the PingCAP VPC, but the data and traffic are encrypted by default. So you do not need to worry about data privacy issues.
 
-## What does the SLA look like?
+## Support FAQs
```
Comment (TomShawn) [inline tidb-cloud/tidb-cloud-faq.md:120]:
```suggestion
## Support FAQ
```

File: tidb-cloud/tidb-cloud-faq.md [hunk 6]
```diff
@@ -69,42 +67,58 @@ TiKV is the transactional store used to store OLTP data. All the data in TiKV is
...
+
+## Migration FAQ
 
-## Is there an easy migration path from another RDBMS to TiDB Cloud?
+### Is there an easy migration path from another RDBMS to TiDB Cloud?
 
 TiDB is highly compatible with MySQL. You can migrate data from any MySQL-compatible databases to TiDB smoothly, whether the data is from a self-hosted MySQL instance or RDS service provided by the public cloud. For more information, see [Migrate Data from MySQL-Compatible Databases](/tidb-cloud/migrate-data-into-tidb.md).
 
-## How does TiDB protect data privacy and ensure security?
+## Feature FAQs
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-faq.md:92]:
```suggestion
## HTAP FAQs
```

File: tidb-cloud/tidb-cloud-faq.md [hunk 7]
```diff
@@ -69,42 +67,58 @@ TiKV is the transactional store used to store OLTP data. All the data in TiKV is
...
+### How does TiDB replicate data between the TiKV nodes?
 
 TiKV divides the key-value space into key ranges, and each key range is treated as a “Region”. In TiKV, data is distributed among all nodes in a cluster and uses the Region as the basic unit. PD is responsible for spreading (scheduling) Regions as evenly as possible across all nodes in a cluster.
 
 TiDB uses the Raft consensus algorithm to replicate data by Regions. Multiple replicas of a Region stored in different nodes form a Raft Group.
 
 Each data change is recorded as a Raft log. Through Raft log replication, data is safely and reliably replicated to multiple nodes of the Raft Group.
 
-## How do I make use of TiDB Cloud’s HTAP capabilities?
+## High Availability FAQ
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-faq.md:78]:
```suggestion
## High availability FAQ
```

File: tidb-cloud/tidb-cloud-faq.md [hunk 8]
```diff
@@ -69,42 +67,58 @@ TiKV is the transactional store used to store OLTP data. All the data in TiKV is
...
 
 No. TiDB Cloud runs on the PingCAP VPC, but the data and traffic are encrypted by default. So you do not need to worry about data privacy issues.
 
-## What does the SLA look like?
+## Support FAQ
 
-TiDB Cloud provides 99.99% SLA. For details, see [Service Level Agreement for TiDB Cloud Services](https://en.pingcap.com/legal/service-level-agreement-for-tidb-cloud-services/).
+### What support is available for customers?
+
+TiDB Cloud is supported by the same team behind TiDB, which has run mission-critical use cases for over 1500 global enterprises across industries including financial services, e-commerce, enterprise applications, and gaming. Support is available 24/7 for TiDB Cloud users.
```
Comment (lilin90) [inline tidb-cloud/tidb-cloud-faq.md:124]:
```suggestion
TiDB Cloud is supported by the same team behind TiDB, which has run mission-critical use cases for over 1500 global enterprises across industries including financial services, e-commerce, enterprise applications, and gaming. Support is available 24/7 for TiDB Cloud customers.
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Review by lilin90 — APPROVED** [review]
LGTM

**Pull request merged**